### PR TITLE
[Feature][Jobschedule] add jobschedule  interface

### PR DESCRIPTION
**Pull request by yangyunxi**
timer, list interface, offline interface, create interface

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] Change does not need document change

### CODE REVIEW

**Review by an-shi-chi-fan — COMMENTED** [review]
LGTM

File: datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/DayCronImpl.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    @Override
+    public String getFuncName(){
+        return "";
+    }
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
+        StrategyFactory.register("day", this);
```
Comment (zixi0825) [inline datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/DayCronImpl.java:66]: day 你可以在getFuncName 中返回，然后在afterPropertiesSet用

File: datavines-server/src/main/java/io/datavines/server/coordinator/repository/service/impl/JobScheduleServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    @Override
+    public List<String> create(JobScheduleCreate jobScheduleCreateCreate) throws DataVinesServerException {
+        String cron = "";
+        List<String> listCron=new ArrayList<String>();
+        try {
+            JobSchedule jobSchedule = new JobSchedule();
+            BeanUtils.copyProperties(jobScheduleCreateCreate, jobSchedule);
+            String type = jobScheduleCreateCreate.getType();
+            MapParam param=jobScheduleCreateCreate.getParam();
+            ObjectMapper mapper = new ObjectMapper();
```
Comment (zixi0825) [inline datavines-server/src/main/java/io/datavines/server/coordinator/repository/service/impl/JobScheduleServiceImpl.java:65]: 使用JSONUtils 去处理

File: datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/HourCronImpl.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                .withHour(always())
+                .withMinute(on(Integer.parseInt(minute)))
+                .withSecond(on (0))
+                .instance();
+
+        return cron.asString();
+    }
+
+    @Override
+    public String getFuncName(){
```
Comment (zixi0825) [inline datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/HourCronImpl.java:59]: 同上

File: datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/MonthCronImpl.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                .withHour(on(Integer.parseInt(hour)))
+                .withMinute(on(Integer.parseInt(mintute)))
+                .withSecond(on (0))
+                .instance();
+
+        return cron.asString();
+    }
+
+    @Override
+    public String getFuncName(){
```
Comment (zixi0825) [inline datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/MonthCronImpl.java:60]: 同上

File: datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/NhourCronImpl.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+                .withDoM(always())
+                .withHour(every(on(hour),nhour))
+                .withMinute(on(mintute))
+                .withSecond(on (0))
+                .instance();
+        return cron.asString();
+    }
+
+    @Override
+    public String getFuncName(){
```
Comment (zixi0825) [inline datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/NhourCronImpl.java:59]: 同上

File: datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/NminuteCronImpl.java [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+                .withDoM(always())
+                .withHour(always())
+                .withMinute(every(on(minute),nminute))
+                .withSecond(on (0))
+                .instance();
+        return cron.asString();
+    }
+
+    @Override
+    public String getFuncName(){
```
Comment (zixi0825) [inline datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/NminuteCronImpl.java:58]: 同上

File: datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/WeekCronImpl.java [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+                .withHour(on(hour))
+                .withMinute(on(minute))
+                .withSecond(on(0))
+                .instance();
+
+        return cron.asString();
+    }
+
+    @Override
+    public String getFuncName(){
```
Comment (zixi0825) [inline datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/WeekCronImpl.java:63]: 同上

Comment (yangyunxi) [reply datavines-server/src/main/java/io/datavines/server/coordinator/server/quartz/cron/impl/WeekCronImpl.java:63]: see #49

**Pull request closed without merging**